sympy/geometry/polygon.py

```diff
@@ -438,6 +438,32 @@ def second_moment_of_area(self, point=None):
         return I_xx, I_yy, I_xy
 
 
+    def polar_modulus(self):
+        """Returns the polar modulus of a two-dimensional polygon"""
+        return self.second_moment_of_area()[0] + self.second_moment_of_area()[1]
+
+
+    def section_modulus(self, point=None):
+        """Returns a tuple with the section modulus of a two-dimensional
+        polygon.
+        """
+        x_c, y_c = self.centroid.x, self.centroid.y
+        if point is None:
+            # taking x and y as maximum distances from centroid
+            x_min, y_min, x_max, y_max = self.shape.bounds
```

Comment:
I am not aware of a `shape` attribute. Do you just mean `self.bounds`?

Reply:
Yes, it is actually `shape.bounds`. I mistakenly got this from other file